### ISSUE TITLE
Create styles file with reusable styles and measurements

### ISSUE THREAD

**Issue by AlbertoCasasOrtiz** [opensim-org/opensim-viewer#86]
For example, the AppBar at the top I think is '68px' (In desktop, I am not sure about phones), and I have to hardcode it every time I use it. It would be useful to have a way of making this "importable", like in android where you can have a common styles file for these cases.

This will also allow to have standard values for height/width/displacements in components across the application.

This issue would supose a big change, since we would have to go across all of the files looking for hardcoded values and fix it.

**Comment by aymanhab** [comment]
@AlbertoCasasOrtiz Is this still a valid open issue? Thanks

**Comment by AlbertoCasasOrtiz** [comment]
I don't think so, as the values that are hardcoded right now are standard (like 68px for top bars), and we introduced the css files for GUI and app versions.
